opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -318,6 +318,12 @@ def __init__(
     def attributes(self) -> types.Attributes:
         return self._attributes
 
+    @property
+    def dropped_attributes(self) -> int:
```

Comment:
Does `Link` s need to have this property as well?

Reply:
Yes, Link now extends `Attributed`